tagevent.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What is the file for`?

Reply:
this should have been used for local testing of the github action, i suppose its not needed in repo

Reply:
Is now deleted